.azure-pipelines/runner.yml

```diff
@@ -165,14 +165,14 @@ jobs:
     inputs:
       command: build
       configuration: $(buildConfiguration)
-      arguments: -l:DatadogLogger,"$(DD_DOTNET_TRACER_MSBUILD)"
+      arguments: -l:DatadogLogger,"$(OTEL_DOTNET_TRACER_MSBUILD)"
```

Comment:
The prefix rename should focus on the env vars exposed to the users, I don't think it is the case of this one.

Reply:
Anyway, if there are only a few of those then it makes sense to rename them now.

Reply:
I would leave these changed if you don't mind

Reply:
NP.